src/github_services.py

```diff
@@ -199,6 +199,8 @@ def get_pull_request_dict_with_timestamp(
     """
 
     for assignee in pr_dict['assignees']:
+        if event['assignee'] is None or assignee is None:
```

Comment:
@kevintab95 Could you please add a comment explaining the circumstances here before we merge this?

Reply:
Update -- I went ahead and made the change.